pages/health-beta/index.md

```diff
@@ -1,11 +0,0 @@
----
-title: Your VA.gov Account
-layout: page-react.html
-entryname: health-beta
-private: true
----
-<nav class="va-nav-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list" role="menubar" aria-label="Primary">
-    <li><a href="/">Home</a></li>
-  </ul>
-</nav>
```

Comment:
This app was not migrated, because there is no application with entryname `health-beta`.